ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -15,5 +26,63 @@ def get_classification(self, image):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        #TODO implement light color prediction
+        if not self.graph_created:
+            # TODO: Check how to safely include this model file when buliding catkin package.
+            self.create_detection_graph("./frozen_inference_graph.pb")
+            return
+
+        traffic_light_index = self.infer(image)
+        TLClassifier.convert_tl_id(traffic_light_index)
         return TrafficLight.UNKNOWN
```

Comment:
get_classification() always returns unknown state?

Reply:
Should the function return the result of convert_tl_id from line 35?

Reply:
yes